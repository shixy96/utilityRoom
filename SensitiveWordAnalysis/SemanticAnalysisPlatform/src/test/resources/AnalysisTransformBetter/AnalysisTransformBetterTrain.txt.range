x
-1 1
1 0 59.940000000000005
2 0 32.400000000000006
3 0 49.799999999999997
4 0 68.039999999999992
5 0 73.900000000000006
6 0 162.68400000000003
7 0 247.24000000000007
8 0 159.40800000000002
9 0 328.89952000000011
10 0 300.4700400000001
11 0 281.5230360000001
12 0 116.60400000000001
13 0 145.98660000000004
14 0 142.911
15 0 110.88090000000003
16 0 108.86400000000003
17 0 102.51000000000002
18 0 81
19 0 157.95000000000002
20 0 3
21 0 21.899999999999999
