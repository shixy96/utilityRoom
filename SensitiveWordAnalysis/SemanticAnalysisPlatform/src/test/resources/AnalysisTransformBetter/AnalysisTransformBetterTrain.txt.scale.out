log2c=5 log2g=-7 rate=77.1678
log2c=-1 log2g=-7 rate=74.5969
log2c=5 log2g=-1 rate=83.1995
log2c=-1 log2g=-1 rate=78.9293
log2c=11 log2g=-7 rate=79.3091
log2c=11 log2g=-1 rate=85.2599
log2c=5 log2g=-13 rate=74.5969
log2c=-1 log2g=-13 rate=74.5969
log2c=11 log2g=-13 rate=76.8627
log2c=-3 log2g=-7 rate=74.5969
log2c=-3 log2g=-1 rate=76.9686
log2c=-3 log2g=-13 rate=74.5969
log2c=5 log2g=1 rate=85.1105
log2c=-1 log2g=1 rate=83.0314
log2c=11 log2g=1 rate=86.8721
log2c=-3 log2g=1 rate=79.8444
log2c=9 log2g=-7 rate=78.5932
log2c=9 log2g=-1 rate=85.1043
log2c=9 log2g=-13 rate=74.4289
log2c=9 log2g=1 rate=86.4862
log2c=5 log2g=-11 rate=74.5658
log2c=-1 log2g=-11 rate=74.5969
log2c=11 log2g=-11 rate=77.5475
log2c=-3 log2g=-11 rate=74.5969
log2c=9 log2g=-11 rate=76.9063
log2c=3 log2g=-7 rate=74.3666
log2c=3 log2g=-1 rate=81.9981
log2c=3 log2g=-13 rate=74.5969
log2c=3 log2g=1 rate=84.9424
log2c=3 log2g=-11 rate=74.5969
log2c=5 log2g=-5 rate=78.4625
log2c=-1 log2g=-5 rate=74.5285
log2c=11 log2g=-5 rate=81.1765
log2c=-3 log2g=-5 rate=74.5969
log2c=9 log2g=-5 rate=80.0249
log2c=3 log2g=-5 rate=77.3483
log2c=15 log2g=-7 rate=82.5335
log2c=15 log2g=-1 rate=86.2807
log2c=15 log2g=-13 rate=78.0766
log2c=15 log2g=1 rate=87.4074
log2c=15 log2g=-11 rate=78.7302
log2c=15 log2g=-5 rate=82.7575
log2c=5 log2g=-15 rate=74.5969
log2c=-1 log2g=-15 rate=74.5969
log2c=11 log2g=-15 rate=74.0865
log2c=-3 log2g=-15 rate=74.5969
log2c=9 log2g=-15 rate=74.5596
log2c=3 log2g=-15 rate=74.5969
log2c=15 log2g=-15 rate=78.6928
log2c=-5 log2g=-7 rate=74.5969
log2c=-5 log2g=-1 rate=74.5845
log2c=-5 log2g=-13 rate=74.5969
log2c=-5 log2g=1 rate=77.0433
log2c=-5 log2g=-11 rate=74.5969
log2c=-5 log2g=-5 rate=74.5969
log2c=-5 log2g=-15 rate=74.5969
log2c=5 log2g=3 rate=87.9552
log2c=-1 log2g=3 rate=85.484
log2c=11 log2g=3 rate=87.7498
log2c=-3 log2g=3 rate=83.7099
log2c=9 log2g=3 rate=87.7498
log2c=3 log2g=3 rate=87.1833
log2c=15 log2g=3 rate=87.6937
log2c=-5 log2g=3 rate=81.2636
log2c=7 log2g=-7 rate=78.0517
log2c=7 log2g=-1 rate=83.8904
log2c=7 log2g=-13 rate=74.5596
log2c=7 log2g=1 rate=86.0753
log2c=7 log2g=-11 rate=74.4351
log2c=7 log2g=-5 rate=79.1783
log2c=7 log2g=-15 rate=74.5969
log2c=7 log2g=3 rate=87.8182
log2c=5 log2g=-9 rate=74.4227
log2c=-1 log2g=-9 rate=74.5969
log2c=11 log2g=-9 rate=78.3318
log2c=-3 log2g=-9 rate=74.5969
log2c=9 log2g=-9 rate=77.5599
log2c=3 log2g=-9 rate=74.5596
log2c=15 log2g=-9 rate=79.39
log2c=-5 log2g=-9 rate=74.5969
log2c=7 log2g=-9 rate=76.9437
log2c=1 log2g=-7 rate=74.5534
log2c=1 log2g=-1 rate=80.3486
log2c=1 log2g=-13 rate=74.5969
log2c=1 log2g=1 rate=83.7971
log2c=1 log2g=-11 rate=74.5969
log2c=1 log2g=-5 rate=74.4787
log2c=1 log2g=-15 rate=74.5969
log2c=1 log2g=3 rate=86.5546
log2c=1 log2g=-9 rate=74.5969
log2c=5 log2g=-3 rate=80.2179
log2c=-1 log2g=-3 rate=76.3025
log2c=11 log2g=-3 rate=83.2991
log2c=-3 log2g=-3 rate=74.4974
log2c=9 log2g=-3 rate=82.8509
log2c=3 log2g=-3 rate=79.1285
log2c=15 log2g=-3 rate=85.0047
log2c=-5 log2g=-3 rate=74.5969
log2c=7 log2g=-3 rate=81.3757
log2c=1 log2g=-3 rate=78.0579
log2c=13 log2g=-7 rate=81.3321
log2c=13 log2g=-1 rate=85.9135
log2c=13 log2g=-13 rate=77.4728
log2c=13 log2g=1 rate=87.451
log2c=13 log2g=-11 rate=78.3754
log2c=13 log2g=-5 rate=82.4588
log2c=13 log2g=-15 rate=77.3607
log2c=13 log2g=3 rate=87.7871
log2c=13 log2g=-9 rate=78.7053
log2c=13 log2g=-3 rate=84.0274
